### DISCUSSION TITLE
Tracee on Android

### DISCUSSION THREAD

**Discussion by MartinaMorcos** [aquasecurity/tracee#353] [Questions and Help] [answered]
Hello, Is it possible to use Tracee on a rooted Android phone?

**Comment by itaysk** [comment]
@yanivagman this one's for you :)

**Comment by yanivagman** [comment, marked as answer]
Hi @MartinaMorcos !

Tracee doesn't support ARM64 architecture yet, so currently this is not possible.
We do have plans to add support for this in the future (#234 ).

**Reply by yanivagman** [reply]
One more thing...

As this is the subject of my master thesis (android malware detection with ebpf) - you can be sure that this support will be added soon :-)

**Reply by MartinaMorcos** [reply]
Awesome, I will keep an eye out on the new update :)

**Reply by itaysk** [reply]
@MartinaMorcos FYI, ARM support was just merged in with https://github.com/aquasecurity/tracee/pull/371, although it's not released yet, you can try to experiment with it.

**Reply by MartinaMorcos** [reply]
That's great, thank you. I will definitely try it out.
